### PR TITLE
Redis bot

### PR DESCRIPTION
**Pull request by pedromreis**
Redis bot impementation, outputs to a defined redis server, db  and queue

### CODE REVIEW

**Comment by sebix** [comment]
As I pointed out in one of your numerous previous pull requests, please add tests for the output bot.

**Comment by pedromreis** [comment]
@sebix not forgotten, but since there is no test example for output bots, I'm trying to understand how can I build one. If I understand correctly, the test will look at the output queue to compare? Loaded from pipeline right?
I need to see this with more time. But I'm it.

**Comment by sebix** [comment]
> @sebix <https://github.com/sebix> not forgotten, but since there is no
> test example for output bots, I'm trying to understand how can I build
> one. If I understand correctly, the test will look at the output queue
> to compare? Loaded from pipeline right?
> I need to see this with more time. But I'm it.
>
The bots needs to write to db 10 (used for all tests), and then you
check get the data with the redis-libs and compare the data for equality.

File: intelmq/tests/bots/outputs/redis/test_output.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+                 "extra.non_ascii": "ççãããã\x80\ua000 \164 \x80\x80 abcd \165\166",
+                 "raw": "eyJ0cm9qYW5mYW1pbHkiOiJTYWxpdHlwMnAiLCJlbnYiOnsic"
+                 "mVtb3RlX2FkZHIiOiIxNTIuMTY2LjExOS4yIiwicmVtb3RlX3"
+                 "BvcnQiOiI2NTExOCIsInNlcnZlcl9hZGRyIjoiNTIuMTguMTk"
+                 "2LjE2OSIsInNlcnZlcl9wb3J0IjoiOTc5NiJ9LCJfdHMiOjE0"
+                 "NjExMDc3NjgsIl9nZW9fZW52X3JlbW90ZV9hZGRyIjp7ImNvd"
+                 "W50cnlfbmFtZSI6IkRvbWluaWNhbiBSZXB1YmxpYyJ9fQ=="
+                 }
+
+EXAMPLE_OUTPUT = {"classification.type": "malware",
```
Comment (sebix) [inline intelmq/tests/bots/outputs/redis/test_output.py]: This is exactly the same as EXAMPLE_EVENT. Why define it two times?

**Comment by pedromreis** [comment]
hummm, no need now really, useful when testing. I'll remove that,

**Comment by sebix** [comment]
Merged via 268bc88a153d462682703060d1c1f4e01a99c08e

**Pull request closed without merging**